build/grunt.js

```diff
@@ -437,6 +450,12 @@ module.exports = function(grunt) {
       }
     },
     shell: {
+      babel: {
+        command: 'npm run babel -- --w',
```

Comment:
this runs our npm babel script (a simple babel command) with the `--w` flag for watch. When a file changes, just that one will be compiled.